# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../zaptel/split_make_install.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- zaptel-1.4.1/Makefile.orig	2007-03-30 14:16:27.000000000 +0000
+++ zaptel-1.4.1/Makefile	2007-03-30 14:20:53.000000000 +0000
@@ -212,7 +212,7 @@
 all: menuselect.makeopts 
 	@$(MAKE) _all
 
-_all: $(if $(BUILD_MODULES),modules) programs $(LTZ_SO) $(LTZ_A)
+_all: programs $(LTZ_SO) $(LTZ_A)
 
 xpp-utils:
 ifeq (yes,$(BUILD_XPP))
@@ -386,7 +386,7 @@
 $(UTILSO): %.o: %.c
 	$(CC) $(CFLAGS) -o $@ -c $<
 
-install: all devices install-modules install-libs install-include install-xpp
+install: devices install-libs install-include install-xpp
 ifeq ($(HOTPLUG_FIRMWARE),yes)
 	$(MAKE) -C firmware hotplug-install
 endif
@@ -403,12 +403,7 @@
 	fi
 	$(INSTALL) -m 644 doc/ztcfg.8 $(INSTALL_PREFIX)/usr/share/man/man8
 	$(INSTALL) -m 644 doc/zttool.8 $(INSTALL_PREFIX)/usr/share/man/man8
-	[ `id -u` = 0 ] && /sbin/depmod -a $(KVERS) || :
 	[ -f $(CONFIG_FILE) ] || $(INSTALL) -D -m 644 zaptel.conf.sample $(CONFIG_FILE)
-	build_tools/genmodconf $(BUILDVER) "$(ROOT_PREFIX)" "$(filter-out zaptel ztdummy xpp zttranscode ztdynamic,$(BUILD_MODULES)) $(MODULE_ALIASES)"
-	@if [ -d /etc/modutils ]; then \
-		/sbin/update-modules ; \
-	fi
 
 install-libs: $(LTZ_SO) $(LTZ_A)
 	$(INSTALL) -D -m 755 $(LTZ_A) $(LIB_DIR)/$(LTZ_A)
